_config.yml

```diff
@@ -1041,11 +1041,11 @@ vendors:
   # fastclick: //cdnjs.cloudflare.com/ajax/libs/fastclick/1.0.6/fastclick.min.js
   fastclick:
 
-  # Internal version: 1.9.7
-  # See: https://github.com/tuupola/jquery_lazyload
+  # Internal version: 1.10.0
```

Comment:
version: 1.9.0

Reply:
? It is 1.10.0

Reply:
![image](https://user-images.githubusercontent.com/15902347/60937574-7d10e780-a303-11e9-82d3-0c4226441945.png)
😂 ???

Reply:
I created https://github.com/theme-next/theme-next-lazyload, and I'm quite sure I am using 1.10.0

Reply:
jsDelivr is also using this version..

Reply:
The internal version is 1.10.0, I just copy and paste files from 1.10.0 folder

Reply:
![image](https://user-images.githubusercontent.com/15902347/60937872-6dde6980-a304-11e9-9bac-8d8a3b1398d5.png)

Reply:
So what... you will get a 404 not found if you use 1.10.0 here.